Jenkinsfile

```diff
@@ -21,7 +17,18 @@ pipeline {
         }
         stage('Unit Tests') {
             steps {
-                sh 'build/run make -j\$(nproc) check'
+                sh 'build/run make -j\$(nproc) test'
+            }
+        }
+        stage('Integration Tests') {
+            environment {
+                KUBE_VERSION = "v1.6"
+                SUITE = "BlockStorage"
+            }
+            steps {
+                sh 'env | sort'
+                sh 'tests/scripts/kube-dind-cluster.sh up'
+                sh 'build/run make -j\$(nproc) test-integration'
```

Comment:
is this kick off all tests under /tests dir? we want to kick off only smoke test right, should this be ' sh 'build/run make -j\$(nproc) test-integration SmokeSuite'